src/commands/lambda/function.ts

```diff
@@ -40,6 +40,46 @@ export interface InstrumentationSettings {
   tracingEnabled: boolean
 }
 
+const MAX_LAMBDA_STATE_CHECKS = 3
+
+/**
+ * Waits for n ms
+ * @param ms
+ * @returns
+ */
+const wait = (ms: number): Promise<void> => new Promise((res) => setTimeout(res, ms))
+
+/**
+ *
```

Comment:
Is this doc comment just here for linting reasons? Seems redundant with the types?

Reply:
Bad editor config, need to remove jsdoc for `.ts`